### PR TITLE
fix(TropykusBorrowing): lend and borrow in ERC20 tests

### PR DESCRIPTION
**Pull request by davidfragalaureano**
Adds ERC20 tests for Borrowing service when the collateral and borrow are both ERC20 tokens. 

`DOC` and `RIF` are the tokens used for these tests.

### CODE REVIEW

File: test/services/TropykusBorrowingService.spec.ts [hunk 1]
```diff
@@ -114,7 +115,7 @@ describe('Tropykus Borrowing Service', () => {
       tropykusBorrowingService.connect(externalWallet).borrow(
         mtx,
         ethers.utils.parseEther(amountToBorrow.toString()),
-        0, // Not in use for now
+        listingId, // Not in use for now
```
Comment (dvrosalesm) [inline test/services/TropykusBorrowingService.spec.ts:118]: Are these comments still revelant?

File: test/services/TropykusBorrowingService.spec.ts [hunk 2]
```diff
@@ -282,7 +283,7 @@ describe('Tropykus Borrowing Service', () => {
           .borrow(
             mtx,
             ethers.utils.parseEther(amountToBorrow.toString()),
-            0, // Not in use for now
+            listingId, // Not in use for now
```
Comment (dvrosalesm) [inline test/services/TropykusBorrowingService.spec.ts:286]: Same question for these

File: test/services/TropykusBorrowingService.spec.ts [hunk 3]
```diff
@@ -882,23 +879,17 @@ describe('Tropykus Borrowing Service', () => {
...
-            ethers.constants.AddressZero
-          );
-        const borrowTxReceipt = await tx.wait();
-        const borrowTxCost = toSmallNumber(
-          calculateTxGasInRBTC(borrowTxReceipt)
+        await tropykusBorrowingService.connect(externalWallet).borrow(
+          mtx,
+          ethers.utils.parseEther(amountToBorrow.toString()),
+          listingId,
+          0, // Not in use for now
```
Comment (dvrosalesm) [inline test/services/TropykusBorrowingService.spec.ts:886]: Same for this

File: test/services/TropykusBorrowingService.spec.ts [hunk 4]
```diff
@@ -1081,49 +1051,37 @@ describe('Tropykus Borrowing Service', () => {
...
-            0, // Not in use for now
-            ethers.constants.AddressZero
-          );
-        const borrowTxReceipt = await tx.wait();
-        const borrowTxCost = calculateTxGasInRBTC(borrowTxReceipt);
+        await tropykusBorrowingService.connect(externalWallet).borrow(
+          mtx,
+          ethers.utils.parseEther(amountToBorrow.toString()),
+          listingId,
+          0, // Not in use for now
```
Comment (dvrosalesm) [inline test/services/TropykusBorrowingService.spec.ts:1058]: Same for this

**Review by dvrosalesm — APPROVED** [review]
LG!

File: test/services/TropykusBorrowingService.spec.ts (hunk 1 above)
Comment (davidfragalaureano) [reply test/services/TropykusBorrowingService.spec.ts]: Nope, removed :)

File: test/services/TropykusBorrowingService.spec.ts (hunk 2 above)
Comment (davidfragalaureano) [reply test/services/TropykusBorrowingService.spec.ts]: Removed.

File: test/services/TropykusBorrowingService.spec.ts (hunk 3 above)
Comment (davidfragalaureano) [reply test/services/TropykusBorrowingService.spec.ts]: Removed.

File: test/services/TropykusBorrowingService.spec.ts (hunk 4 above)
Comment (davidfragalaureano) [reply test/services/TropykusBorrowingService.spec.ts]: Removed.

**Pull request merged**